addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -23,6 +24,19 @@
         return _.find(slideList, slideMatch);
     };
 
+    /**
+     * Helper: Return all the slides matching the given criteria (we might have 2 slides
+     * with the same slide ID, because of "isQuiz" set to True / False)
+     *
+     * @private
+     * @param {Array<Object>} slideList List of dict reprensenting a slide
+     * @param {Object} matcher (see https://underscorejs.org/#matcher)
+     */
+    const findSlides = function (slideList, matcher) {
+        const slideMatch = _.matcher(matcher);
+        return _.filter(slideList, slideMatch);
+    };
```

Comment:
I added a parameter `returnAll` to the existing function, tell me what you think :)
(if true, return all slides matching the criteria instead of just the first one)